src/CMakeLists.txt

```diff
@@ -14,12 +14,23 @@ find_package(LibtorrentRasterbar REQUIRED)
 list(APPEND QBT_QT_COMPONENTS Core Network Xml)
 if (GUI)
     list (APPEND QBT_QT_COMPONENTS Concurrent Gui Widgets)
+    if (APPLE)
+        list (APPEND QBT_GUI_OPTIONAL_LINK_LIBRARIES objc)
+        list (APPEND QBT_QT_COMPONENTS MacExtras)
+    endif (APPLE)
 endif (GUI)
 if (DBUS)
     list (APPEND QBT_QT_COMPONENTS DBus)
 endif (DBUS)
 find_package(Qt5 5.5.1 COMPONENTS ${QBT_QT_COMPONENTS} REQUIRED)
 
+if (GUI AND APPLE)
+    include_directories(${Qt5MacExtras_INCLUDE_DIRS})
```

Comment:
I thought this myself, but actually it does not work. If I remove these lines, compilation will stop working.
I could not explain the reason and simply hardcoded the variables. I think it is a bug in Qt, because I saw other people doing similar things on the net.